frontend/.env.example

```diff
@@ -0,0 +1 @@
+BACKEND_ENDPOINT=http://localhost:8080
```

Comment:
For the `.env.example` would this normally be something like =BACKEND_URL ?

Reply:
I think putting localhost is fine, as it helps suggest what the value would be for local development. I would agree if it was any other sort of sensitive information, like keys or passwords